gradle/sonar.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'org.sonarqube'
+
+// disable sonar for pull request builds until
+// sonar plugin for pull request comments is available at sonarcloud.io
+if (!new Boolean(System.env.TRAVIS_PULL_REQUEST)) {
+    rootProject.tasks['sonarqube'].setEnabled(false)
```

Comment:
I disable sonar in pull requests intentionally
Default analysis mode analyses whole project and uploads analysis into sonar pretenting it to be the current state of the project.
So pull requests are usually anlysed in 'preview' mode along with a special sonar plugin commenting found issues in pull request. Alas, sonarcloud.io does not have such a plugin so pull requests analysis is useless. That is why I disable it